src/main/java/no/nav/fo/veilarbregistrering/db/arbeidssoker/ArbeidssokerperioderMapper.java

```diff
@@ -37,6 +36,26 @@ static Arbeidssokerperioder map(List<Formidlingsgruppeendring> formidlingsgruppe
                     .filter(Formidlingsgruppeendring::erAktiv)
                     .collect(toList());
 
+    private static Function<List<Formidlingsgruppeendring>, List<Formidlingsgruppeendring>> slettTekniskeISERVEndringer =
+            (formidlingsgruppeendringer -> {
+
+                int j = 0;
+
+                while (j < formidlingsgruppeendringer.size() - 1) {
+                    if (formidlingsgruppeendringer.get(j).getFormidlingsgruppeEndret().equals(formidlingsgruppeendringer.get(j+1).getFormidlingsgruppeEndret())) {
```

Comment:
Jeg synes denne løkka blir litt tung å forholde seg til, men samtidig har jeg ikke noe bedre selv :)

Tenker det kan være fint å merge dette inn, også evt. iterere på det etterpå. Funksjonen er jo veldig fin, og gjør jobben.

Reply:
Enig :/ Jeg fant heller ikke noe bedre. Prøvde først med en `for`-loop men det synes jeg var enda verre enn dette.